CONTRIBUTING.md

```diff
@@ -39,7 +39,6 @@ pre-commit install
 
 With help of pre-commit, your future commits to this project will be reformatted with our black configuration,
 which includes settings `skip-string-normalization = true` and `max-line-length = 79`.
-On the command-line, our configuration would be equivalent to `black -S -l 79 -t py36 --exclude vendored\|_vispy .`.
```

Comment:
Why remove this? I think it's very useful for other libraries considering adding black. (such a thing was proposed at scikit-image a few weeks ago)

Reply:
I removed it because we have to update it every time we change our `black` config and right now our config is super long (we exclude like 10 different things)

Reply:
How about I add a reference to the file where these settings are stored instead?

Reply:
Besides, it's not our job to document black; if people are interested in using it, they should look at the official documentation instead.

Reply:
adding a reference to the file where this info is stored sounds like a reasonable compromise to me, especially as the `exclude` list might continue to change

Reply:
Great, I'm into the reference idea.